dynawo/sources/Solvers/AlgebraicSolvers/test/TestBasics.cpp

```diff
@@ -130,7 +130,7 @@ TEST(AlgebraicSolvers, testInit) {
 TEST(AlgebraicSolvers, testModifySettings) {
   boost::shared_ptr<Model> model = initModelFromDyd("dyd/solverTestAlpha.dyd");
   boost::shared_ptr<SolverKINAlgRestoration> solver(new SolverKINAlgRestoration());
-  ASSERT_NO_THROW(solver->init(model, SolverKINAlgRestoration::KIN_NORMAL, 1, 1, 1, 1, 1, 1, 1));
+  ASSERT_NO_THROW(solver->init(model, SolverKINAlgRestoration::KIN_YPRIM, 1, 1, 1, 1, 1, 1, 1));
```

Comment:
No more algebraic variables there?

Reply:
No, as y has become a calculated variable, and x is a derivative one

Reply:
This was caused by the issue with mode change in calculated variables. Solved.